File: soal8.pas
program Soal8;
uses crt;
var
    pixel: integer;
begin
    clrscr;
    write('Masukkan nilai pixel: ');
    readln(pixel);

    if pixel > 255 then
        pixel := 255
    else if pixel < 0 then
        pixel := 0;

    writeln('Nilai pixel setelah clipping = ', pixel);
    readln;
end.

File: soal8.c
#include <stdio.h>

int main() {
    int pixel;

    printf("Masukkan nilai pixel: ");
    scanf("%d", &pixel);

    if (pixel > 255)
        pixel = 255;
    else if (pixel < 0)
        pixel = 0;

    printf("Nilai pixel setelah clipping = %d\n", pixel);

    return 0;
}